build.gradle

```
@@ -60,10 +63,14 @@ jar {
 
 dependencies {
     api platform('com.exactpro.th2:bom:4.2.0')
-    implementation 'com.exactpro.th2:common:5.2.0-dev'
-    implementation 'com.exactpro.th2:codec:5.2.0-dev'
+    implementation 'com.exactpro.th2:common:5.3.0-json-raw-body-5144283961-5ae02cd-SNAPSHOT'
+    implementation 'com.exactpro.th2:codec:5.2.0-new-proto-5111859401-e24c7b8-SNAPSHOT'
+
+    implementation 'io.netty:netty-buffer:4.1.93.Final'
 
     implementation "org.slf4j:slf4j-api"
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.2'
```

```suggestion
    implementation 'org.apache.commons:commons-lang3'
    implementation 'com.fasterxml.jackson.core:jackson-databind'
```
th2-bom provides versions. If you want to update versions could you please change bom project